src/Pages/Authentication/Login.js

```diff
@@ -1,7 +1,75 @@
+import { useState } from 'react'
+import { ToastContainer , toast} from 'react-toastify'
+import { useAuth } from '../../Context/AuthContext'
+ import 'react-toastify/dist/ReactToastify.css';
+ import './Auth.css'
+ import { useNavigate } from 'react-router-dom'
+
 export const Login = () =>{
+    const navigate = useNavigate()
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const { handleLoginData} = useAuth()
+    const [error, setError] = useState(false)
+    const [message, setMessage] = useState('')
+    const test = {
+        email: "adarshbalika@gmail.com",
+        password: "adarshBalika123"
+    }
```

Comment:
As Akanksha suggested, please use different credentials for your test user. Other than adarshbalika